Aptoide/aptoide-client-v8
# [APP-776] Notify me

**What does this PR do?**

This PR aims at implementing the cancel flow for the notify me button on the app coming soon card.

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

Press the Notify me button, check that you receive the notification. Repeat the flow, but press the cancel button after pressing the notify me button and check that you DO NOT receive the notification.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-776](https://aptoide.atlassian.net/browse/APP-776)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
